plugins/filter/decrypt.py

```diff
@@ -18,7 +109,7 @@
 
 def decrypt_filter(data, input_type='yaml', output_type='yaml', sops_binary='sops', rstrip=True, decode_output=True,
                    aws_profile=None, aws_access_key_id=None, aws_secret_access_key=None, aws_session_token=None,
-                   config_path=None, enable_local_keyservice=None, keyservice=None):
+                   config_path=None, enable_local_keyservice=False, keyservice=None):
```

Comment:
I don't know what this does but it seems to be a strange change to include in this PR.

From the looks of it, I guess that it does not (currently) change behavior, it's just "more correct"?

Reply:
It is indeed strange! I mainly noticed that I'm using the doc fragment here (well of course before I wasn't because there was no documentation, but the intention was to implement it), but had the wrong default value for this option. Since it is a boolean option and it only checks for truthy values (https://github.com/ansible-collections/community.sops/blob/main/plugins/module_utils/sops.py#L92, https://github.com/ansible-collections/community.sops/blob/main/plugins/module_utils/sops.py#L73) `None` and `False` are equivalent, so this is no behavior change.